source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -69,6 +69,17 @@ content: |
                    secretAccessKey: "<IAM User Secret Access Key>",
                }
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+
+           "kmsProviders" = {
+               "aws": {
+                   "access_key_id":  "<IAM User Access Key ID>",
+                   "secretAccessKey": "<IAM User Secret Access Key>",
```

Comment:
I did not check the Python API, but am wondering whether `secretAccessKey` is correct or should it be snake case?